Makefile

```diff
@@ -14,3 +14,52 @@ compose_up:
 
 compose_down:
 	docker-compose -f docker/docker-compose.yml down
+
+# Development Setup
+.venv:
+	python3 -m venv .venv
+	. .venv/bin/activate
+	python3 -m pip install -r requirements-dev.txt
+
+.git/hooks/pre-commit: .venv
+	python3 -m pre_commit install
+
+.env:
+	# afterwords, set in .env
+	# TAKAHE_DATABASE_SERVER="postgres://postgres:insecure_password@localhost:5433/takahe"
+	cp development.env .env
+
+_PHONY: python-dev
+python-dev: .venv
+
+_PHONY: precommit
+precommit: python-dev .git/hooks/pre-commit
```

Comment:
Why is this `python-dev`?

Reply:
Sorry switched from `python-dev` to `pydev` halfway through hashtag-adhd

Reply:
Done